cmd/jimmctl/cmd/migratemodel.go

```diff
@@ -20,15 +17,24 @@ import (
 
 const (
 	migrateModelCommandDoc = `
-The migrate command migrates a model(s) to a new controller. Specify
-a model-uuid to migrate and the destination controller name.
+The migrate commands migrates a model, or many models between two controllers
+registered within JIMM. 
+
+You may specify a model name (of the form owner/name) or model UUID.
+
+For example, JIMM is aware of controller-a and controller-b and controller-a has
+models:
+	- alice@canonical.com/model-a
+	- 2cb433a6-04eb-4ec4-9567-90426d20a004
+
+You may run the migrate command like so to migrate the two models
+from controller-a to controller-b:
+	jimmctl migrate controller-b model-a 2cb433a6-04eb-4ec4-9567-90426d20a004
```

Comment:
Examples should be left out of the explanation doc and just left for the examples

Reply:
I like this example and want to keep it, but putting it in examples might look a bit strange as it will display like:

```
Examples:

You may specify a model name (of the form owner/name) or model UUID.

For example, JIMM is aware of controller-a and controller-b and controller-a has
models:
	- alice@canonical.com/model-a
	- 2cb433a6-04eb-4ec4-9567-90426d20a004

You may run the migrate command like so to migrate the two models
from controller-a to controller-b:
	jimmctl migrate controller-b model-a 2cb433a6-04eb-4ec4-9567-90426d20a004
```

Reply:
I suggest keeping the explanation in the doc and then just move the example.
```
You may specify a model name (of the form owner/name) and/or model UUID.
Examples:
	jimmctl migrate myController myModel myOtherModel
        jimmctl migrate myController 2cb433a6-04eb-4ec4-9567-90426d20a004 34b433a6-04eb-4ec4-9567-90426d20a005
```

Reply:
Done